manifests/plugins/iostat.pp

```diff
@@ -36,9 +38,23 @@
     require => Package['sysstat']
   }
 
-  collectd::plugins::plugin_common { 'iostat':
-    package_name         => 'collectd-python',
-    plugin_file_name     => '10-iostat.conf',
-    plugin_template_name => 'iostat/10-iostat.conf.erb',
+  file { 'get iostat_types.db':
+    ensure  => present,
+    replace => 'yes',
+    path    => '/usr/share/collectd/iostat-collectd-plugin/iostat_types.db',
+    owner   => root,
+    group   => 'root',
+    mode    => '0755',
+    content => template($db_template),
+  }
+
+  collectd::plugin { 'iostat':
+    package_name        => 'collectd-iostat',
```

Comment:
This package doesn't exists and should either be removed or changed to collectd-python.  RHEL platforms fail on this.